bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -69,6 +69,63 @@ spec:
             cpu: 100m
             memory: 300Mi
 
+      # ovn-nbctl daemon 
+      - name: ovn-nbctl
+        image: {{.OvnImage}}
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -e
+          if [[ -f /env/_master ]]; then
+            set -o allexport
+            source /env/_master
+            set +o allexport
+          fi
+          ovn_config_namespace=openshift-ovn-kubernetes
+          retries=0
+          while true; do
```

Comment:
we're talking to the unix domain socket - we don't need the endpoint.

Reply:
OK